students/biegon_piotr/lesson_05_list_comprehensions/age_calculator.py

```diff
@@ -0,0 +1,20 @@
+avg = 0
+ages = []
+people = int(input("Enter the number of people: "))
+
+for x in range(people):
+    print("Person number", x + 1, "age: ")
+    ages.append(int(input()))
+
+adults = [x for x in ages if x >= 18]
+kids = [x for x in ages if x < 18]
+
+if adults == []:
+    print("No adults")
+else:
+    for i in adults:
+        avg += i
+    avg /= len(adults)
```

Comment:
the less iterations and for loops the better. I would advise to use here instead:
`avg = sum(adults) / len(adults)`